make/e2e-setup.mk

```diff
@@ -187,8 +187,8 @@ E2E_SETUP_OPTION_BESTPRACTICE_HELM_VALUES_FILE ?= $(BINDIR)/scratch/values-bestp
 $(E2E_SETUP_OPTION_BESTPRACTICE_HELM_VALUES_FILE): | $(BINDIR)/scratch
 	$(CURL) $(E2E_SETUP_OPTION_BESTPRACTICE_HELM_VALUES_URL) -o $@
 
-## Dependencies which will be added to e2e-setup-certmanager depending on the
-## supplied E2E_SETUP_OPTION_ variables.
+# Dependencies which will be added to e2e-setup-certmanager depending on the
+# supplied E2E_SETUP_OPTION_ variables.
```

Comment:
This is a drive-by fix to prevent this internal comment showing up in the `make help` output.